build_projects.yml

```diff
@@ -7,6 +7,8 @@ pr:
 
 variables:
 - group: noos_releases_group_variables
+- name: isMain
+  value: $[eq(variables['Build.SourceBranch'], 'refs/heads/main')]
```

Comment:
Do we currently have a `main` branch?

`refs/head/main` -> `refs/head/master` ?

Also a general comment: we should support in *.yml both `master` and `main` to avoid possible future issues.

Reply:
changed to master, copy paste error